src/main/java/frc/robot/subsystems/swerve/Converstions.java

```diff
@@ -20,6 +20,10 @@ public static double degreesToFalcon(double degrees, double gearRatio) {
         return degrees / (360.0 / (gearRatio * COUNTS_PER_REV_FALCON));
     }
 
+    public static double RPMToDegrees(double degrees, double gearRatio) {
+        return falconToDegrees(RPMToDegrees(degrees, gearRatio), gearRatio);
+    }
```

Comment:
Everyone, run away as fast as possible and take shelter!!! This is recursion without a stopping condition!